pages/terms_of_service.mdx

```diff
@@ -0,0 +1,19 @@
+# Terms of Service
+
+These are our terms of service. In case you have any questions or concerns please reach out to us.
+
+## Refunds
+
+You registration is non-refundable once purchased. Cancellations of your travel and/or hotel reservations are your own responsibility. Please notice, that if you can't attend for any reason you are responsible to cancel your room and flights.
+
+## Transferring your Ticket
+
+You can transfer your ticket until the day before the conference. Transferring your ticket once the conference has started is not allowed.
+
+## Cancellation
+
+The organisers can change the date of the event or cancel the event. If the event is cancelled, any purchased ticket gets a full refund. If only a part e. g. a workshop is not taking place you get back the fee for the booked part, excluding the fee for the parts you still attend. Further claims against the organiser are not possible.
+
+## Media
+
+Our team will record videos and take photos at the event. It may happen that you will be included. With your registration you grant us the right to use such material taken over the days. Still everyone has the change during registration to inform our team that she or he doesn't like to be photographed. We will do our best to not take any pictures of you. Regardless if you informed our team about your preference in case you have concerns about any photo or video snippet that we published please let us know and we will remove it.
```

Comment:
> Still everyone has the change during registration to inform

`change` -> `chance` but I'd rewrite the whole as:

> If you don't want to be photographed, please inform us at the day of the conference at the registration desk.

Let's remove `We will do our best to not take any pictures of you.`

@nikgraf I'm not sure how we'll track these. Concat had different lanyards for such requests.